docs/preview/features/correlation.md

```diff
@@ -148,7 +148,16 @@ public class OrderController : ControllerBase
 
 ## Logging
 
-As an additional feature, we provide an extension to use the HTTP correlation directly in a [Serilog](https://serilog.net/) configuration:
+As an additional feature, we provide an extension to use the HTTP correlation directly in a [Serilog](https://serilog.net/) configuration as an [enricher](https://github.com/serilog/serilog/wiki/Enrichment). 
+This adds the correlation information of the current request to the log event as a log property called `TransactionId` and `OperationId`.
+
+**Example**
+
+- `TransactionId`: `A5E90591-ADB0-4A56-818A-AC5C02FBFF5F`
+- `OperationId`: `79BB196A-B0CC-4F5C-B48A-AB87850346AF`
+
+**Usage**
+The enricher requires access to the application services so it can get the correlation information.
```

Comment:
Great to have the docs centralized.  I'm just wondering if we should mention here something regarding that `CorrelationInfoEnricher`, or is that now something that is obsolete ?

Reply:
It's not obsolete, but we added an extension for better dev experience. So, the 'old' way is still valid, but not recommended via the docs. We could maybe say something like: ... adds a `CorrelationInfoEnricher` to the Serilog enrichment... or something like that. But not sure if this would provide much added-value, as we should maybe guide people to the extension described here.

Reply:
I didn't get that from just reviewing :)